Update doc for decoupling of remote cluster state with remote backed data storage (#6730)
opensearch-project/documentation-website: _tuning-your-cluster/availability-and-recovery/remote-store/remote-cluster-state.md

The cluster state metadata is managed by the elected cluster manager node and is essential for the cluster to properly function. When the cluster loses the majority of the cluster manager nodes permanently, then the cluster may experience data loss because the latest cluster state metadata might not be present in the surviving cluster manager nodes. Persisting the state of all the cluster manager nodes in the cluster to remote-backed storage provides better durability.

When the remote cluster state feature is enabled, the cluster metadata will be published to a remote repository configured in the cluster.

> Any time new cluster manager nodes are launched after disaster recovery, the nodes will automatically bootstrap using the latest metadata stored in the remote repository. This way you get durability of metadata. 

Reviewer: ```suggestion
Any time new cluster manager nodes are launched after disaster recovery, the nodes will automatically bootstrap using the latest metadata stored in the remote repository. This provides metadata durability. 
```

> Note that remote cluster state can be enabled independent of remote backed data storage.

Reviewer: ```suggestion
You can enable remote cluster state independently of remote-backed data storage.
{: .note}

```

> If durability of data is required then the remote backed data storage should also be enabled as specified in [remote store documentation]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/remote-store/index/).

Reviewer: ```suggestion
If you require data durability, you must enable remote-backed data storage as described in the [Remote store documentation]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/remote-store/index/).
```

The cluster state metadata is managed by the elected cluster manager node and is essential for the cluster to properly function. When the cluster loses the majority of the cluster manager nodes permanently, then the cluster may experience data loss because the latest cluster state metadata might not be present in the surviving cluster manager nodes. Persisting the state of all the cluster manager nodes in the cluster to remote-backed storage provides better durability.

When the remote cluster state feature is enabled, the cluster metadata will be published to a remote repository configured in the cluster.
Any time new cluster manager nodes are launched after disaster recovery, the nodes will automatically bootstrap using the latest metadata stored in the remote repository. This provides metadata durability. 

> You can enable remote cluster state independently of remote-backed data storage.

Reviewer 2: Should "the" precede "remote"?

{: .note}

> If you require data durability, you must enable remote-backed data storage as described in the [Remote store documentation]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/remote-store/index/).

Reviewer 2: ```suggestion
If you require data durability, you must enable remote-backed data storage as described in the [remote store documentation]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/remote-store/index/).
```
